test/staking.ts

```diff
@@ -193,7 +193,7 @@ export const stakingTests = (): void => {
 
             const registrationTypes = [RegistrationTypes.OnChain];
             await patronIam.createClaimRequest({
-                claim: { claimType: `${patronRole}.${root}`, claimTypeVersion: 1, fields: [] },
```

Comment:
I don't understand why this improves the test? Is it because it is unrealistic to have an empty array for the `fields`?